src/Swarm/TUI/Controller.hs

```diff
@@ -224,9 +222,8 @@ handleMainEvent s = \case
     s' <- toggleModal s CommandsModal
     continue (s' & gameState . availableCommands . notificationsCount .~ 0)
   ControlKey 'g' -> case s ^. uiState . uiGoal of
```

Comment:
Can we just call `toggleModal` instead of having a `case`?

Reply:
Ah, no, that would be different: the point is that `Ctrl-G` does nothing if there is no current goal.